python/requirements.txt

```diff
@@ -4,6 +4,7 @@ numpy==1.24.2
 pandas==1.5.3
 scipy==1.10.0
 PyYAML==5.4.1
+urllib3<2.0  # Issue -> https://stackoverflow.com/questions/76187256/importerror-urllib3-v2-0-only-supports-openssl-1-1-1-currently-the-ssl-modu
```

Comment:
do you now why did not have it before ?

Reply:
I think the issue is coming from internal PyPi. Someone probably uploaded urllib3>2.0 recently and now it is auto installing that latest version.

Reply:
No, it was installing 2.1.0 earlier too. I am confused now as well.